updating/updating_a_cluster/updating-cluster-rhel-compute.adoc

```diff
@@ -13,15 +13,15 @@ To do: Remove this comment once 4.13 docs are EOL.
 ////
 
 You can update, or upgrade, an {product-title} cluster. If your cluster contains
-Red Hat Enterprise Linux (RHEL) machines, you must perform more steps to update
+{op-system-base-full} machines, you must perform more steps to update
```

Comment:
This isn't a result of your changes, but we should avoid hard-wrapping lines. Do you mind condensing these three lines into a single one?

Reply:
Also if it's okay with you, I think I'll remove "or upgrade" at the beginning since it's not really valuable, and our glossary is in favor of [update](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update) anyways.